Srclib.Nuget/ConsoleCommands/DepresolveConsoleCommand.cs

```diff
@@ -10,124 +10,135 @@
+            return allDeps;
+        }
+
+
+        public static async Task<IEnumerable<LibraryDescription>> DepResolve(Project proj)
         {
-          var context = new ApplicationHostContext
-          {
-            Project = proj,
-            TargetFramework = f
-          };
-
-          return ApplicationHostContext.GetRuntimeLibraries(context).Skip(1); // the first one is the self-reference
-        })
-        .Distinct(LibraryUtils.Comparer)
-        .OrderBy(l => l.Identity?.Name);
-
-    static async Task RunResolve(string dir)
-    {
-      var p = new Process();
-      p.StartInfo.WorkingDirectory = dir;
-      p.StartInfo.FileName = _dnuPath.Value;
-      p.StartInfo.Arguments = "restore";
-      p.StartInfo.UseShellExecute = false;
-      p.StartInfo.CreateNoWindow = true;
-      p.StartInfo.RedirectStandardInput = true;
-      p.StartInfo.RedirectStandardOutput = true;
-      p.StartInfo.RedirectStandardError = true;
-
-      p.Start();
-      // it's important to read stdout and stderr, else it might deadlock
-      var outs = await Task.WhenAll(p.StandardOutput.ReadToEndAsync(), p.StandardError.ReadToEndAsync());
-      p.WaitForExit();
-
-      // in the future, actually parse output or something
-    }
+            await RunResolve(proj.ProjectDirectory);
+            var allDeps = GetAllDeps(proj);
+            return allDeps;
+        }
+
+
+        static IEnumerable<LibraryDescription> GetAllDeps(Project proj) =>
+            proj.GetTargetFrameworks().Select(f => f.FrameworkName)
+            .SelectMany(f =>
+            {
+                var context = new ApplicationHostContext
+                {
+                    Project = proj,
+                    TargetFramework = f
+                };
+                return ApplicationHostContext.GetRuntimeLibraries(context, false).Skip(1); // the first one is the self-reference
+            })
+            .Distinct(LibraryUtils.Comparer)
+            .OrderBy(l => l.Identity?.Name);
+
+
+        static async Task RunResolve(string dir)
+        {
+            var p = new Process();
+            p.StartInfo.WorkingDirectory = dir;
+            p.StartInfo.FileName = _dnuPath.Value;
+            p.StartInfo.Arguments = "restore";
+            p.StartInfo.UseShellExecute = false;
+            p.StartInfo.CreateNoWindow = true;
+            p.StartInfo.RedirectStandardInput = true;
+            p.StartInfo.RedirectStandardOutput = true;
+            p.StartInfo.RedirectStandardError = true;
+            p.Start();
+            // it's important to read stdout and stderr, else it might deadlock
+            var outs = await Task.WhenAll(p.StandardOutput.ReadToEndAsync(), p.StandardError.ReadToEndAsync());
+            p.WaitForExit();
+            // in the future, actually parse output or something
+        }
+
+        static string FindDnuWindows()
+        {
+            return RunForResult(@"C:\Windows\System32\cmd.exe", "/c \"where dnu\"");
```

Comment:
All this dnu activity came from code written by @Alxandr. I kept it intact. I think we can remove anything related to Windows here.